packages/@react-aria/switch/docs/useSwitch.mdx

```diff
@@ -91,6 +91,7 @@ indicate focus. The focus ring is only visible when the user is interacting with
 not with a mouse or touch.
 
 ```tsx example
+import {useSwitch} from '@react-aria/switch';
```

Comment:
this is already included here https://github.com/adobe/react-spectrum/blame/main/packages/%40react-aria/switch/docs/useSwitch.mdx#L22
and it gets surfaced at the top of the docs
![Screen Shot 2021-12-07 at 4 19 09 PM](https://user-images.githubusercontent.com/698229/145126035-082ec130-897f-4d4c-84ed-6b847ef4570a.png)

you'll find this across every doc page

Reply:
you might be able, in this instance, remove it from the line i pointed to, as long as it's in an example, it'll be imported. our docs can only import each one once

Reply:
oh yeah, makes sense. I've now removed that top import so it's only in the example